src/pages/home/index.js

```diff
@@ -95,7 +96,12 @@ const Home = () => {
   } = useForm({ resolver: zodResolver(homeSchema) });
 
   const onSubmit = data => {
-    createTarget({ ...data, ...latLng });
+    if (targetsList.length <= 9) {
+      createTarget({ ...data, ...latLng });
+    } else {
+      setTargetsLimit(true);
+      window.alert(t('target.limit.alert'));
```

Comment:
WDYT if you replace this `alert()`in the future with a modal? Something aesthetic just to improve UX.